client/layout/guided-tours/button-labels.jsx

```diff
@@ -0,0 +1,40 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+
+// Returns React component with a localized label and optional icon
+function button( label, icon ) {
+	return localize( ( { translate: tr } ) => (
+		<strong>
+			{ icon }
```

Comment:
nit: I think its possible to concisely combine both parts of the icon logic:
```jsx
{ icon && [ icon, '\u00A0' ] }
```

Reply:
The trouble with arrays like this is that every element needs a `key`. Otherwise React will complain. I could also use a fragment. But all of that is more complicated than just two JSX expression container next to each other, at least in my view.